commitlint.config.ts

```diff
@@ -76,6 +76,14 @@ function isFooterNote(line: string): boolean {
         isFixesSentence(line);
 }
 
+function includeHashtagRef(text: string) {
```

Comment:
the name of this function suggests that it is procedural (because you're using imperative tense), let's rather replace "include" with "includes" to make it sound more like a func that returns a boolean and doesn't perform any action

Reply:
Done